src/Pyz/Zed/MerchantRelationship/MerchantRelationshipDependencyProvider.php

```diff
@@ -45,4 +49,15 @@ protected function getMerchantRelationshipPostUpdatePlugins(): array
             new ProductListMerchantRelationshipPostUpdatePlugin(),
         ];
     }
+
+    /**
+     * @return array<\Spryker\Zed\MerchantRelationshipExtension\Dependency\Plugin\MerchantRelationshipExpanderPluginInterface>
+     */
+    protected function getMerchantRelationshipExpanderPlugins(): array
```

Comment:
Looks like we have missed the integration of https://spryker.atlassian.net/browse/GLUE-11151
There are two options:
- SKIP: Ensure that it's not blocking the backoffice functionality of MR request feature.
- INTEGRATE (only stable parts - ignore ZAPI modules/plugins): In case it's a blocker.

Reply:
In scope of this PR we don't have to integrate any previously missed features, the feature we are currently integrating works well now